streaming/base/storage/download.py

```diff
@@ -314,6 +277,33 @@ def download_from_azure_datalake(remote: str, local: str) -> None:
         raise
 
 
+def download_from_databricks_unity_catalog(remote: str, local: str, timeout: float) -> None:
+    """Download a file from remote Databricks Unity Catalog to local.
+
+    Args:
+        remote (str): Remote path (Databricks Unity Catalog).
+        local (str): Local path (local filesystem).
+        timeout (float): Timeout in seconds.
+    """
+    from util import (databricks_unity_catalog_credentials_time_remaining,
+                      get_databricks_unity_catalog_volume_info,
+                      set_databricks_unity_catalog_credentials)
+
+    volume_path = remote.replace('uc://', '')
```

Comment:
I think you can use `obj = urllib.parse.urlparse(remote)` which gives you `prefix`, `volume_name` and `file_path`, etc. For reference, you can check out `download.py` file on how it s being used. Thanks!

Reply:
Done